runtime/onert/backend/train/TensorBuilder.h

```diff
@@ -77,6 +90,7 @@ class TensorBuilder
   ir::OperandIndexMap<ir::OperandInfo> _backward_tensor_info_map;
   ir::OperandIndexMap<bool> _as_constants;
   util::Set<DisposableTensorIndex> _disposable_backprops;
+  ir::OperationIndexMap<util::Set<LayerScopeTensorIndex>> _operation_to_layerscope;
```

Comment:
Each op has several LayerScopeTensors.
So, This structures saves `operation_index` - `layerscope_tensor_index`, which will be used for planning.